lib/modules/assets/widgets/assets/video_asset.dart

```diff
@@ -222,26 +224,65 @@ class _VideoAssetState extends State<VideoAsset> {
     }
   }
 
+  String _getVideoInfoText() {
+    final localController = _controller;
+
+    var text = localController != null && _isVideoInitialized && _isVideoReady
```

Comment:
make a const for this line
```suggestion
    var text = should show preview text
```

Reply:
Refactored